lib/adequate_crypto_address/ut.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module AdequateCryptoAddress
+    # 7UT
```

Comment:
I couldn't initialize `7UT`, so `7EVEN Utility Token` is devoid of its proper name atm; how could we go about this?